ParquetClassLibrary/All.cs

```diff
@@ -0,0 +1,226 @@
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test parquets.
+        /// </summary>
+        public static readonly Range<EntityID> BlockIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Sandbox.Parquets.Furnishing"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test parquets.
+        /// </summary>
+        public static readonly Range<EntityID> FurnishingIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Sandbox.Parquets.Collectible"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test parquets.
+        /// </summary>
+        public static readonly Range<EntityID> CollectibleIDs;
+
+        /// <summary>
+        /// A collection containing all defined <see cref="Range{EntityID}"/>s of parquet types.
+        /// </summary>
+        public static readonly List<Range<EntityID>> ParquetIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Sandbox.RoomRecipe"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
+        /// </summary>
+        public static readonly Range<EntityID> RoomRecipeIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Crafting.CraftingRecipe"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
+        /// </summary>
+        public static readonly Range<EntityID> CraftingRecipeIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Quests.Quest"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
+        /// </summary>
+        public static readonly Range<EntityID> QuestIDs;
+
+        /// <summary>
+        /// A subset of the values of <see cref="EntityID"/> set aside for <see cref="Items.Item"/>s.
+        /// Valid identifiers may be positive or negative.  By convention, negative IDs indicate test Items.
+        /// </summary>
+        public static readonly Range<EntityID> ItemIDs;
+        #endregion
+
+        #region Collections
+        /// <summary>
+        /// A collection of all defined parquets of all subtypes.
+        /// This collection is the source of truth about parquets for the rest of the library,
+        /// something like a color palette that other classes can paint with.
+        /// </summary>
+        /// <remarks>All <see cref="EntityID"/>s must be unique.</remarks>
+        public static EntityCollection<ParquetParent> Parquets { get; set; }
```

Comment:
Are collections intentionally not marked with Readonly?

Reply:
Let me double-check~

Reply:
That was in error.  I must have had the old pattern from before the refactor in my head.  Fixed!